packages/@sanity/cli/src/cli.ts

```diff
@@ -76,6 +87,11 @@ export async function runCli(cliRoot: string, {cliVersion}: {cliVersion: string}
     args.groupOrCommand = 'help'
   }
 
+  if (args.groupOrCommand === 'logout') {
+    // flush telemetry events before logging out
+    await flushTelemetry()
```

Comment:
why is this here instead of in the `logoutCommand` code?

Reply:
trying to keep the interface exposed to subcommands minimal, so not passing the `flush` method to subcommands at this point. We might change this later if we see that it's something we want subcommands to be able to do, but for now it made sense to special case this here.